### PR TITLE
Activity Log: Put a page title on this page to match others.

### PR DESCRIPTION
**Pull request by beaulebens**
Part of #22869 

I've just copy-pasted how it's done elsewhere in this file. I'm not familiar enough with how things work to know how to use `DocumentHead` as suggested :)

This is probably mergeable, but we should figure out and replace all similar instances with `DocumentHead` throughout this file.

### CODE REVIEW

**Comment by beaulebens** [comment]
Note the current code does get the page title to display properly.

screen shot 2018-03-05 at 3 55 16 pm

File: client/my-sites/stats/controller.jsx [hunk 1]
```diff
@@ -446,6 +446,9 @@ export default {
 		if ( siteId && ! isJetpack ) {
 			page.redirect( '/stats' );
 		} else {
+			// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+			context.store.dispatch( setTitle( i18n.translate( 'Stats', { textOnly: true } ) ) );
```
Comment (dmsnell) [inline client/my-sites/stats/controller.jsx]: How about using `<DocumentHead>` instead? See the [comments section](https://github.com/Automattic/wp-calypso/blob/37b89f4cc9e1514a4186a2796752831ce987766c/client/my-sites/comments/main.jsx#L82) for an example.

Do you have time? If not, I can make this change for you. I'd just prefer we don't willingly introduce obviated code, especially when it's easy not to.

Comment (beaulebens) [reply client/my-sites/stats/controller.jsx]: I actually am probably not going to get to doing this properly, so if you can handle it, I'd really appreciate it @dmsnell, thanks. I'll keep an eye out on here and be happy to test though!

**Comment by dmsnell** [comment]
Updated @beaulebens :+1: while at it I updated the other instances of `setTitle()` to make the file consistent and I cleaned up a few things to simplify some logic, remove mutation, and introduce insignificant performance improvements by preventing some additional object clones. If the cleanup is too much I can back out that commit.

**Comment by beaulebens** [comment]
Thanks @dmsnell and @blowery !

File: client/my-sites/stats/controller.jsx (hunk 1 above)
Comment (vindl) [reply client/my-sites/stats/controller.jsx]: Just a note that a switch to `DocumentHead` is causing some issues with the title here (try switching to `Weeks` for example). On the other hand, the good ol' obviated code is working as expected. :) I logged an issue https://github.com/Automattic/wp-calypso/issues/23924 for it, hopefully we'll be able to get to it soon.

**Pull request merged**